ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOtherDirectionHeaderType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
+
+public enum CCDOtherDirectionHeaderType {
+    HEADER_UPLOAD,
```

Comment:
HEADER word seems redundant. can you please remove the word HEADER in front of UPLOAD and CONFIRM ? Thanks

Reply:
Unfortunately I can't, because that's used in the CCD UI - in the def file we have something like "otherDirectionHeaders=\"HEADER_*\""}, I had the same comment but realised it's not possible :(